charts/federation-v2/README.md

```diff
@@ -118,7 +118,7 @@ chart and their default values.
 | controllermanager.clusterHealthCheckSuccessThreshold | Minimum consecutive successes for the cluster health to be considered successful after having failed.                                                                                        | 1                                                                                                     |
 | controllermanager.clusterHealthCheckTimeoutSeconds   | Number of seconds after which the cluster health check times out.                                                                                                                            | 3                                                                                                     |
 | clusterregistry.enabled               | Specifies whether to enable the clusterregistry in federation v2.                                                                                                                                           | true                                                                                                  |
-| global.limitedScope        | Whether the federation namespace will be the only target for federation. If set to true, the value set for `controllermanager.registryNamespace` and `controllermanager.registryNamespace` will be ignored. | false                                                                                                 |
+| global.scope                   | Whether the federation namespace will be the only target for federation. If set to `Namespaced`, the value set for `controllermanager.registryNamespace` and `controllermanager.registryNamespace` will be ignored. | false                                                                                                 |
```

Comment:
Looks like a copy-paste mistake for `controllermanager.registryNamespace`. should have been `controllermanager.federationNamespace`.

Reply:
Currently, if it is set `Namespaced`, only `federationNamespace` is ignored.

Addressed